app/src/main/java/com/preonboarding/sensordashboard/presentation/viewmodel/SensorHistoryViewModel.kt

```diff
@@ -8,17 +8,34 @@ import com.preonboarding.sensordashboard.domain.model.SensorHistory
 import com.preonboarding.sensordashboard.domain.usecase.DeleteSensorHistoryUseCase
 import com.preonboarding.sensordashboard.domain.usecase.GetSensorHistoryListUseCase
 import dagger.hilt.android.lifecycle.HiltViewModel
-import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.collectLatest
+import kotlinx.coroutines.launch
 import javax.inject.Inject
 
 @HiltViewModel
 class SensorHistoryViewModel @Inject constructor(
-    getSensorHistoryListUseCase: GetSensorHistoryListUseCase,
+    private val getSensorHistoryListUseCase: GetSensorHistoryListUseCase,
     private val deleteSensorHistoryUseCase: DeleteSensorHistoryUseCase,
 ) : BaseViewModel() {
-    val sensorHistoryList: Flow<PagingData<SensorHistory>> = getSensorHistoryListUseCase().cachedIn(viewModelScope)
 
-    fun deleteSensorHistory(){
-//        deleteSensorHistoryUseCase()
+    private val _historyList: MutableStateFlow<PagingData<SensorHistory>> =
+        MutableStateFlow<PagingData<SensorHistory>>(PagingData.empty())
+    val historyList: StateFlow<PagingData<SensorHistory>> = _historyList.asStateFlow()
+
+    fun sensorHistoryList() {
+        viewModelScope.launch {
+            getSensorHistoryListUseCase.invoke().cachedIn(viewModelScope).collectLatest {
```

Comment:
invoke()로 접근 안해도 될거에요!
```suggestion
            getSensorHistoryListUseCase().cachedIn(viewModelScope).collectLatest {
```